tests/suites/test_suite_psa_crypto.function

```diff
@@ -8979,7 +8979,7 @@ exit:
 }
 /* END_CASE */
 
-/* BEGIN_CASE depends_on:PSA_WANT_ALG_SHA_256:MBEDTLS_PSA_BUILTIN_ALG_TLS12_PSK_TO_MS */
+/* BEGIN_CASE depends_on:PSA_WANT_ALG_SHA_256:MBEDTLS_PSA_BUILTIN_ALG_TLS12_ECJPAKE_TO_MS */
```

Comment:
I think this should be `MBEDTLS_PSA_BUILTIN_ALG_TLS12_ECJPAKE_TO_PMS`, based on reading `include/mbedtls/config_psa.h`

Reply:
Yes good catch